src/payload-plugin-s3.d.ts

```diff
@@ -1,6 +1,13 @@
 import { UploadedFile } from 'express-fileupload'
+import { Field } from 'payload/types'
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/richardvanbergen/payload-plugin-cloud-storage/pullRequest?prid=7829468)
```suggestion
import { Field } from "payload/types"
```